build.rs.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/build.rs b/build.rs
+index 5aa702e..586f5d8 100644
+--- a/build.rs
++++ b/build.rs
+@@ -45,7 +45,7 @@ fn build_www() {
+     // try `pnpm` first, then `npm`
+     if !www_dir.join("node_modules").exists() {
+         let pnpm_install_success = match Command::new(shell)
+-            .args([shell_flag, "pnpm install"])
++            .args([shell_flag, "pnpm install --offline"])
+             .current_dir(www_dir)
+             .status()
+         {
+@@ -55,7 +55,7 @@ fn build_www() {
+ 
+         if !pnpm_install_success {
+             let npm_install_result = Command::new(shell)
+-                .args([shell_flag, "npm install"])
++                .args([shell_flag, "npm install --offline"])
```

Comment:
ideally the change upstream (for a future release) would be to follow cargo offline mode as set in `CARGO_NET_OFFLINE` env

Reply:
ok i can set that